scripts/call_contracts.py

```diff
@@ -101,7 +101,7 @@ def submit_request(
 
     request_manager = contracts["RequestManager"]
     token = contracts["MintableToken"]
-    total = amount + request_manager.functions.totalFee(amount).call()
+    total = amount + request_manager.functions.totalFee(token.address, amount).call()
```

Comment:
Yeah, this script really accumulated some technical debt, didn't it. Thanks for catching and fixing this!